docs/DeveloperGuide.md

```diff
@@ -404,23 +404,35 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 1.
 
-**Use case: Filter persons by tag**
+**Use case: Find persons by name and/or tag**
 
 **MSS**
 
-1.  User requests to list persons with a specific tag description
+1.  User requests to list persons with a specific name and/or tag
 2.  Coach2K22 shows a list of filtered persons
 
     Use case ends.
 
 **Extensions**
 
-* 1a. The given tag description cannot be found in Coach2K22.
+* 1a. The given name and tag keywords cannot be found in Coach2K22.
 
-    * 1a1. Coach2K22 shows an error message and an empty list.
+    * 1a1. Coach2K22 shows an empty list.
 
       Use case ends.
+    
+* 1b. The keyword provided does not indicate any prefix e.g. `n/` or `t/`.
 
+    * 1b1. Coach2K22 shows an error message.
+
+      Use case ends.
+
+* 1c. No keywords are provided.
+
+    * 1c1. Coach2K22 shows an error message.
+
+      Use case ends.
+    
```

Comment:
I also think it would be better if this part can be more accurate. `find n/Alex t/` is invalid and will show an error message, but it is not strictly "no keywords"?

Reply:
You're right. My original intention for this extension is to deal with the case where the user just enters `find` without specifying the keywords. I guess I should not only make this clearer but also include an extension for `find n/Alex t/`. I will make the amendments and notify you here when changes are made. Thanks!

Reply:
Something like this?
![image](https://user-images.githubusercontent.com/77256455/157607519-1c643b7d-8f5e-4ab0-ac62-7eb784f14dd0.png)

Reply:
LGTM